react/components/admin/AdminStyles.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { PageHeader } from 'vtex.styleguide'
+
+interface Props {
+  title: string
+}
+
+const AdminStyles: React.FC<Props> = ({ children, title }) => (
+  <div className="h-100 min-vh-100 overflow-y-auto bg-light-silver">
+    <div className="center mw8">
+      <PageHeader title={title} />
```

Comment:
⛏ `AdminStyles` is a misleading name for a component that on top of setting the container style also sets the header

Reply:
I see! I'm also not very fond of it, but had a hard time coming up with a better name...

Do you have any suggestions?

Reply:
I've renamed it to `AdminStructure`.